TurkishFilter/sections/specific.txt

```diff
@@ -389,6 +389,12 @@ selamtuzla.com##.aside_ad
 kibriswebhaber.com##img[width="1100"][height="140"]
 petroturk.com##.site-content > div[class^="site-"]
 petroturk.com##div[class^="ads-"]
+unyenethaber.com##div[style^="width: 980px; height: auto; margin: 20px"]
+unyenethaber.com##.haberDetayIcerik > div[style^="margin-right: 10px; float: left;"]
+unyenethaber.com###siteSol > div[style^="width: 780px; height: auto;"]
+unyenethaber.com###siteSag > div[style^="width: 370px; height: auto;"]
+unyenethaber.com###haberAlani > div[style^="width: 100%; height: auto; overflow: hidden;"]
+unyenethaber.com###siteSol > div[style^="width: 728px; height: auto; margin: 20px"]
```

Comment:
It looks like that they changed something again and there is a different ad at the top.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/103276722-9ad67e80-49c7-11eb-918d-9f6c19684866.png)

</details>

There is video player on main page and this rule - `unyenethaber.com###siteSol > div[style^="width: 728px; height: auto; margin: 20px"]` hides it. I'm not sure if this video player should be blocked.

Reply:
> It looks like that they changed something again and there is a different ad at the top.
> 
Yeah, they've added `text-align: center` a few mins ago to `div[style]`